Sumo Stuff/environment_state.py

```diff
@@ -0,0 +1,246 @@
+import traci
```

Comment:
This file contains all the necessary helper scripts. It's probably time to split it or rename it though.